doc_user/UserManual.adoc

```diff
@@ -1946,19 +1946,24 @@ Press *[.underline]##O##K* to finish.
 
 ==== 10.10.6 Close Function (Close)
 
-The close function exits from the *_Therapy window._* Press the *Close* button to access the Close function. The *_Select an option_* *_window_* shown below appears. The *Buttons panel* has the following choices:
+The close function exits from the *_Therapy window._* Press the *Close* button to access the Close function. The *_Select an option_* *_window_* shown below appears. 
+
+The *Buttons panel* has the following choices:
+
+image:image117.png[Therapy save and close,width=379,height=128]
 
 * *[.underline]##Y##es*: this function is the same as the *Save function ([.underline]##S##ave)* described above.
 * *[.underline]##N##o*: return to the *_Therapy_* window without saving the therapy plan.
 * *[.underline]##C##ancel*: exit the function and return to the *_Therapy_* window without applying any changes.
 
-image:image117.png[Therapy save and close,width=379,height=128]
 
 ==== 10.10.7 Notify & SMS 
 
 image:image118.png[Notify & SMS,width=170,height=131]
 
-Use these two checkboxes to set the therapies plan for this patient as important (not implemented yet) and notifiable via SMS. Briefly:
+Use these two checkboxes to set the therapies plan for this patient as important (_[.underline]##not implemented yet##_) and notifiable via SMS. 
+
+Briefly, the SMS notification will work in the following way:
```

Comment:
small improvement,  change `the SMS notification will work in the following way:` to `the SMS notification works in the following way:`

Reply:
Done, thanks!